multiqc/modules/damageprofiler/damageprofiler.py

```diff
@@ -48,6 +48,14 @@ def __init__(self):
 
         if len(self.summary_metrics_data) == 0:
             raise UserWarning
+        
+        # Add all of the data into a single dict
+        self.damage_data = dict()
+        self.damage_data['3PGtoA'] = self.threepGtoAfreq_data
+        self.damage_data['5PCtoT'] = self.fivepCtoTfreq_data
+        self.damage_data['dist_fw'] = self.lgdist_fw_data
+        self.damage_data['dist_rv'] = self.lgdist_rv_data
```

Comment:
Data for plots shouldn't be sent to `self.write_data_file`, only single key:value pairs. Plot data can already be saved to files automatically with `-p` \ `--export` (I've been thinking about making this a default actually).